{{cookiecutter.project_slug}}/Dockerfile

```diff
@@ -19,12 +19,15 @@ RUN pip install pipenv
 # Copy, then install requirements before copying rest for a requirements cache layer.
 COPY Pipfile* /tmp/
 RUN cd /tmp \
-    && pipenv install --system
+    && pipenv install --system --dev
```

Comment:
accidentaly became part of this but meh - a 6 byte PR?

This is so that dev deps are available for dev.